src/frontend/src/test-e2e/register.test.ts

```diff
@@ -231,6 +237,61 @@ test("Log into client application, after registration", async () => {
   });
 }, 300_000);
 
+test("Add key type on login if unknown", async () => {
+  await runInBrowser(async (browser: WebdriverIO.Browser) => {
+    const authenticatorId1 = await addVirtualAuthenticator(browser);
+
+    await browser.url(II_URL);
+    const userNumber = await FLOWS.registerNewIdentityWelcomeView(
+      DEVICE_NAME1,
+      browser
+    );
+
+    const credentials = await getWebAuthnCredentials(browser, authenticatorId1);
+    expect(credentials).toHaveLength(1);
+
+    // check device in II canister
+    const agent = new HttpAgent({
+      host: "http://127.0.0.1:4943",
+      // @ts-ignore
```

Comment:
I'm not exactly sure why the typse don't match. But it works in practice.

Reply:
worth extending `// @ts-ignore with a comment` ?